lib/Command/Scan.php

```diff
@@ -46,6 +46,17 @@ protected function configure() {
 				null,
 				InputOption::VALUE_NONE,
 				'Scan all the group folders.'
+			)
+			->addOption(
+				'path',
+				'p',
+				InputArgument::OPTIONAL,
```

Comment:
This should be `InputOption::VALUE_REQUIRED`.

Reply:
Hi @icewind1991, thanks for your review. Can you explain, why this should be required, if it wasn't before (as it was not existing). If it is not set, we fall back to scan the root of the group folder. In my opinion it is better for backward compatibility to keep it optional, but I wouldn't be against if you argue for "required".

Reply:
I just checked, that for the scan command of the files app, it is the same behaviour and also optional (when omitted also scanning from root). https://github.com/nextcloud/server/blob/master/apps/files/lib/Command/Scan.php#L92

Reply:
(bit late responding here :see_no_evil: )

the "required" part is about value of the option, not the option itself (you can pass just `--path` as option, you have to do `--path /foo`).

`InputArgument::OPTIONAL` just happens to be the same value as `InputOption::VALUE_REQUIRED` by chance so the file scan command works as expected by accident (and should be adjusted).

Reply:
Is there anything one could do to help move this issue forward?